.toolsharerc

```diff
@@ -0,0 +1,5 @@
+pinned_tool:
+  - tool: "improbadoc"
+    version: "2.20.2"
```

Comment:
2.20.5 is being released, you'll need this if you want to use the `LinkTo` macro (for cross linking between repos)